rosdep/base.yaml

```
@@ -795,6 +795,11 @@ emacs:
   gentoo: [virtual/emacs]
   openembedded: [emacs@meta-oe]
   ubuntu: [emacs]
+embree:
+  arch: [embree]
+  debian: [libembree-dev]
```

The debian package seems to only be bullseye (testing) or sid, but the newer debian version supported is `buster` (ros noetic).

IIUC, the debian `entry` should be `null`:

```suggestion
  debian: null
```

---

Rather than an explicit `null` removing the definition is usually preferred and gives the same result. In this case however we could also specify the dependency for the upcoming bullseye distribution.
```suggestion
  debian:
    bullseye: [libembree-dev]
```

Not specifying other distributions will leave them as implicit nulls.

---

Thank you for pointing out this problem, I fixed in https://github.com/ros/rosdistro/pull/28065/commits/09f9beec1ac20d1c155cff070f549bc34258f2fb